
<?xml version="1.0" encoding="utf-8"?>
<launch>

  <!-- Start darknet and ros wrapper -->
  <node pkg="darknet_ros" type="darknet_ros" name="darknet_ros" output="screen" >

    <param name="weights_file" value="$(find darknet_ros)/yolo_network_config/weights/yolov3.weights" />
    <param name="model_file" value="$(find darknet_ros)/yolo_network_config/weights/yolov3.cfg" />
    <rosparam param="detection_classes">[person, dog, bus]</rosparam>


    <param name="detection_threshold" value="0.05" />
    <param name="show_opencv" value="true" />
    <param name="publish_detection_image" value="true" />

    <!--<remap from="camera/image_raw"  to="camera/image_raw" />
    <remap from="~/found_object"  to="~/your_choice_object" />
    <remap from="~/bounding_boxes"  to="~/your_choice_bb" />
    <remap from="~/detection_image"  to="~/your_choice_img" />-->

  </node>

</launch>
